plugins/destination/gcs/client/client_test.go

```diff
@@ -17,9 +17,10 @@ import (
 	"github.com/cloudquery/plugin-sdk/v4/schema"
 	"github.com/rs/zerolog"
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
 )
 
-const bucket = "cq-dest-gcs"
+const bucket = "test-cloudquery"
```

Comment:
Noticing this now, what's the reason to change the bucket name?

Reply:
Sorry, committed this by mistake, I need to set this everytime to run tests, since GCS won't allow me to name my bucket `cq-dest-gcs`since bucket names need to be globally unique in GCS, I was gonna raise this issue on discord to see how we can make testing a bit easier, but haven't had the time to raise it yet.

Reply:
Fixed at https://github.com/cloudquery/cloudquery/compare/abd53a174e96587e90770b0af5469ecf0a793957..2b4bb86f422b2eeaf52046b94623ddc88108aff1

Reply:
I think we can expose it via an environment variable to be configurable (in a separate PR), WDYT?

Reply:
Yes!

Reply:
Opened #17612 , will open a PR for it when I get some time